signac/contrib/filterparse.py

```diff
@@ -18,7 +18,7 @@ def _read_index(project, fn_index=None):
     if fn_index is not None:
         _print_err("Reading index from file '{}'...".format(fn_index))
         fd = open(fn_index)
-        return (json.loads(l) for l in fd)
+        return (json.loads(i) for i in fd)
```

Comment:
```suggestion
        return (json.loads(_) for _ in fd)
```

Reply:
Or `line`, see previous comment.
```suggestion
        return (json.loads(line) for line in fd)
```

Reply:
@mikemhenry Since, for a similar method in `__main__.py`, we have used `line`. Hence, I think I should use `line` here too.

Reply:
Yes `line` is great! My suggestion for `_` was to use it as a throw away variable instead of something like `i` or `l` BUT always always use something descriptive if it makes sense!

Reply:
😊 I distinguish "temporary" variables like `line` that are re-used by name at least once from "throwaway" variables that are never used by name. I use `_` when dealing with functions like [NumPy histograms](https://numpy.org/doc/stable/reference/generated/numpy.histogram.html) that return a tuple like `histogram, bin_edges`. If I don't plan to use bin edges, then I call it like
```python
histogram, _ = np.histogram(...)
```